orchestrator/core/composio/tool_executor.py

```
@@ -342,71 +342,28 @@ def _tokenize(s: str) -> set[str]:
                     if examples:
                         examples_clause = f" Examples of mapped actions: {examples}"
 
-                # Auto-recovery (generic, no hardcoded mappings):
-                # If the requested action isn't mapped, but we can confidently pick a SAFE mapped action
-                # of the same "kind" (read vs write), do it inside this single tool call to avoid
-                # user-visible fail+retry (and avoid duplicate UI steps).
+                # DISABLED: Auto-mapper was using display names from broken sync
+                # Just use the raw action name from LLM and let Composio validate it
                 auto_mapped_from: Optional[str] = None
```

Ruff (0.14.14)

[error] Local variable `auto_mapped_from` is assigned to but never used

Remove assignment to unused variable `auto_mapped_from`

(F841)